modules/grant/annotationProperty.csv

```diff
@@ -6,8 +6,12 @@ Grant Abstract,Abstract for the grant,,,TRUE,,,,,
+Grant Institution Alias,The alias of the institution(s) associated with the grant (e.g. UCSD). 1...*,"ASU, AU, BCM, BIDMC, BMC, BU, BWH, Barrow, CHOP, COH, CSHL, Caltech, Cleveland Clinic, Columbia, Cornell, DFCI, Dartmouth, Duke, Emory, GTech, HMS, HUJI, Harvard, Houston Methodist, ISB, IU, IUPUI, JHU, Jackson Laboratory, Lurie Children's Hospital, MGH, MIT, MSKCC, Mayo, Moffitt, NU, NYU, OHSU, PNNL, Pitt, Rockefeller University, SBU, Sage, Salk, StJude, Stanford, U Miami, UA, UAB, UCBerkeley, UCD, UCI, UCLA, UCMC, UCSD, UCSF, UChicago, UD, UFL, UIC, UMMS, UMichigan, UMinn, UNC, UNR, UNSW, UPenn, USC, UT Austin, UT Southwestern, UTHSCSA, UUtah, UVA, UW, UWM, University of Illinois, University of Texas MD Anderson Cancer Center, Vanderbilt, WUSTL, WCM, Wake Forest, Wistar, Yale",,TRUE,,,,,list like
 Grant Investigator,Investigator(s) associated witht the grant. 1...*,,,TRUE,,,,,list like
 Grant Consortium Name,Consortium(s) associated with the grant. 1...1,"CCBIR, CSBC, HTAN, ICBP, MetNet, NCI, NCI Clinical and Translational Exploratory/Developmental Studies, PDMC, PS-ON, Sage Bionetworks, TEC",,TRUE,,,,,
 GrantView_id,A unique primary key that enables record updates using schematic.,,,TRUE,,,,,unique
+Grant Start Date,The start date of the grant YYYY-MM-DD format,,,TRUE,,,,,date
+NIH RePORTER Link,Link to the search results for this grant number on the NIH Reporter website,,,TRUE,,,,,url
+Duration of Funding,"Duration of the funding period, in years",,,FALSE,,,,,
```

Comment:
For duration of funding and embargo, I would add "int" as the validation rule ([rules are defined on this page](https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/2645262364/Data+Model+Validation+Rules]))